src/pages/config/config-details-changes.jsx

```diff
@@ -28,7 +28,7 @@ export function ConfigDetailsChangesPage() {
         setIsLoading(false);
       });
 
-    getConfig(id)
+    getConfigName(id)
```

Comment:
I think we should extract this into a ConfigBreadcrumb component + add the config icon (with a tooltip for type)